src/js/views/patients/schedule/schedule-list.scss

```diff
@@ -52,7 +52,7 @@
 }
 
 .schedule-list__day-list-row {
-  cursor: default;
+  cursor: pointer;
 
   &:hover {
     background: $black-90;
```

Comment:
I would assume the background on hover is blue and this grey when mult-selected

Reply:
Ooh, good point. I forgot about that feature of bulk edit. I'll confirm with Mark.